Bottom: 44bfde289ab26aeb4198759eb1b330a2100c5203
Top:    d2941ddf88827b63fb51872194cb7d31b6dd157d
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-12 09:18:12 +0800

Refresh of high-performance-mode

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 4eb608c..894f781 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1891,7 +1891,7 @@ static int tun_set_iff(struct net *net, struct file *file, struct ifreq *ifr)
 
 		tun = netdev_priv(dev);
 		tun->dev = dev;
-		tun->flags = flags | IFF_TX_RING;
+		tun->flags = flags;
 		tun->txflt.count = 0;
 		tun->vnet_hdr_sz = sizeof(struct virtio_net_hdr);
